Jenkinsfile_CNP

```diff
@@ -14,8 +14,14 @@ def channel = '#bsp-build-notices'
 GradleBuilder builder = new GradleBuilder(this, product)
 
 def secrets = [
+  's2s-${env}': [
+    secret('microservicekey-bulk-scan-orchestrator', 'S2S_SECRET')
```

Comment:
do we need to load it from s2s vault ? Can we not copy secret into bulk scanning vault and load it from there ?

Reply:
Also where are we going to use those secrets in functional tests application yaml or main application yaml

Reply:
Not sure I can copy the secret from s2s vault - it does not allow me to access it

Reply:
All these secrets are used in both main and func tests application.yaml

Reply:
This one is for functional tests only. They create test exception records directly in CCD.

Reply:
Worth making a TODO to change it to payment processor when possible

Reply:
> Can we not copy secret into bulk scanning vault and load it from there ?

all secrets are already copied and ready to use. iirc this was one of the steps/prerequisites for migrating to AKS

Reply:
Please update this as this is incorrect. Refer slack channel for details